developing/README.md

```diff
@@ -0,0 +1,196 @@
+## Setting up a full dev environment
+
+If you want to work on Proxstar using a 1:1 development setup, there are a couple things you're going to need
+
+- A machine you can SSH into, portforward from, and run Flask, Redis, and Docker on
+- At least one (1) Proxmox host running Proxmox >6.3
+- Docker
+- SSH portforwarding
+- A CSH account
+- An RTP (to tell you secrets)
+
+1. Configure your Proxmox node
+
+I would recommend setting up a development account on your Proxmox node. Name it anything. (Maybe `proxstartest`?). This is necessary to grab authentication tokens and the like. It should have the same permissions as `root@pam`. You can accomplish this by creating a group in `Datacenter > Permissions > Groups` and adding `Administrator` permissions to the group, then adding your user to the group. If you do this, then it's easy to enable/disable it for development. You should also generate an SSH key for the user.
+
+You will also have to set up a pool on your Proxmox node with your csh username. To do this, go into `Datacenter > Permissions > Pools > Create`.
```

Comment:
How do pools get created on real proxstar? A bit pedantic, but capitalise CSH?

Reply:
Good catch on the `CSH` lol.

Reply:
Can confirm. This is how it works. I'll still include the instructions, but I'll mention that it'll auto-create the pool.